pvlib/tests/test_pvarray.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+from numpy.testing import assert_allclose
+
+from pvlib import pvarray
+
+
+def test_pvefficiency_adr():
+    g = [1000, 200, 1000, 200, 1000, 200, 0, np.nan]
+    t = [25, 25, 50, 50, 75, 75, 25, 25]
+    params = [1.0, -6.651460, 0.018736, 0.070679, 0.054170]
+
+    eta = [1.0, 0.949125, 0.928148, 0.876472, 0.855759, 0.803281, 0.0, np.nan]
```

Comment:
Can we add a comment mentioning the source of the known result?

Reply:
Sure.